xtable-utilities/pom.xml

```diff
@@ -90,33 +88,6 @@
            <scope>compile</scope>
        </dependency>
 
-        <dependency>
-           <groupId>org.apache.parquet</groupId>
-           <artifactId>parquet-avro</artifactId>
-        </dependency>
-
-        <!-- AWS Dependencies -->
-        <dependency>
-            <groupId>org.apache.hadoop</groupId>
-            <artifactId>hadoop-aws</artifactId>
-        </dependency>
-        <dependency>
-            <groupId>com.amazonaws</groupId>
-            <artifactId>aws-java-sdk-bundle</artifactId>
-        </dependency>
-
-        <!-- Azure Dependencies -->
-        <dependency>
-            <groupId>org.apache.hadoop</groupId>
-            <artifactId>hadoop-azure</artifactId>
-        </dependency>
-
-        <!-- GCS Dependencies -->
-        <dependency>
-            <groupId>com.google.cloud.bigdataoss</groupId>
-            <artifactId>gcs-connector</artifactId>
-        </dependency>
```

Comment:
removing these for the same reason as above.

Reply:
Without these jars, user won't be able to run the xtable jar sync for s3 or gcs paths ?